canonical-web-and-design/ubuntu.com
# Creates /osm page

## Done

- Creates /osm page

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/telco/osm
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Check modal form works and that the static form works is JS is disbaled.
- Compare against [copy-doc](https://docs.google.com/document/d/1LQim0WRVdDZE9-ToFKSD52O7kgurMoytrgDDAPYnug8/edit)


## Issue / Card

Fixes https://github.com/canonical-web-and-design/ubuntu.com/issues/10064
